src/routes/index.svelte

```diff
@@ -39,102 +43,219 @@
+            opacity: 0;
+        }
+
+        60% {
+            opacity: 0.8;
+        }
+
+        100% {
+            opacity: 1;
+        }
     }
 </style>
 
 <svelte:head>
     <title>NUS Angklung Ensemble</title>
 </svelte:head>
 
-<img src="/homepage/header.svg" alt="NUS Angklung Ensemble" width="50%" />
+<img src="client/home/header.svg" alt="NUS Angklung Ensemble" width="50%" />
 <br />
-<p>Listen to the sound of Nusantara</p>
+<p>Listen to the sounds of NUSantara</p>
 <button class="navigation-button">Join Us Now</button>
 <button class="navigation-button">Learn To Play</button>
 <br />
 <div class="angklung-logo">
     <img
         class="big-angklung"
-        src="/homepage/angklung-large.svg"
+        src="client/home/angklung-large.svg"
         alt="Angklung-large"
         width="20%" />
     <img
         class="mid-angklung"
-        src="/homepage/angklung-mid.svg"
+        src="client/home/angklung-mid.svg"
         alt="Angklung-mid"
         width="18%" />
-    <img src="/homepage/angklung-small.svg" alt="Angklung-small" width="16%" />
+    <img
+        src="client/home/angklung-small.svg"
+        alt="Angklung-small"
+        width="16%" />
 </div>
 
 <div class="message">
-    <h2>LOREM IPSUM DOLOR SIT AMET</h2>
+    <h2 class="tagline">Kudu silih asih, silih asah, jeung silih asuh</h2>
     <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
-        convallis neque ac ante lacinia vestibulum. Suspendisse ligula tortor,
-        egestas ac tincidunt nec, finibus vel orci. Maecenas gravida commodo
-        felis ut tincidunt.
+        Sundanese for “to remind one another, to care for one another, and to
+        guide one another” - is the mantra we adhere to. Unity in angklung
+        subsequently gives birth to what we call our ensemble as a “famsemble” -
+        a portmanteau of family and ensemble.
     </p>
+    <a href="vision-mission">Learn more about our vision and mission</a>
 </div>
 
 <div class="deck">
     <div class="card">
-        <h3>
-            <img src="/homepage/music-note.svg" alt="music-note" width="23px" />
-            OUR MUSIC
-        </h3>
+        <div class="card-title">
+            <img src="client/home/music-note.svg" alt="music-note" />
+            <h3>Our Music</h3>
+        </div>
         <p>
-            We play a diverset set of song, ranging from contemporary to
-            traditional songs. Lorem ipsum dolor sit amet, consectetur
-            adipiscing elit. Curabitur convallis neque ac ante lacinia
-            vestibulum.
+            NUS Angklung Ensemble showcases angklung’s versatility that balances
+            traditionality and modernity through performing pieces that cater to
+            both traditional interests and current popular trends.
         </p>
-        <a href="our-music">Learn more about what we play >>></a>
+        <div class="card-action">
+            <a href="our-music">What we play</a>
+            <svg
```

Comment:
May I know what is this?

Reply:
It is the "->" svg that slides in when you hover